sample/db/samples/reimbursements.rb

```diff
@@ -2,21 +2,43 @@
 
 Spree::Sample.load_sample("orders")
 
-order          = Spree::Order.last
-inventory_unit = order.inventory_units.first
+order = Spree::Order.last
+inventory_unit = order.inventory_units.take!
```

Comment:
Hey @kennyadsl I think we're not handling exceptions when this `takes!` method fails, should we still use the bang method?

Reply:
Ah nevermind, I just saw spaghetticode's comment and that solved my doubt here